Source/Cells/BrickCell.swift

```diff
@@ -106,6 +111,15 @@ open class BaseBrickCell: UICollectionViewCell {
     open override func layoutSubviews() {
         super.layoutSubviews()
         brickBackgroundView?.frame = self.bounds
+
+        if frame.width == requestedWidth {
+            self.layoutIfNeeded() // This layoutIfNeeded is added to make sure that the subviews are laid out correctly
```

Comment:
It won't. The reason it's there is that when the view is laid out, the subviews aren't at this case. If we remove this line, the unit tests will fail